frontend/src/organisms/Burger.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { route } from 'src/Routes';
+import { NavLink, Button } from 'src/atoms/';
+import { useAuth } from 'src/utils/auth';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
+export function Burger () {
```

Comment:
tohle bude na mobilu chtít trochu zvětšit

![image](https://user-images.githubusercontent.com/29277166/143680691-ee54b85c-8fb5-41f8-87f3-a62b96f6a5f3.png)